README.md

```diff
@@ -22,20 +22,52 @@ Afterwards clone the repo and install it:
 This will install `osmtogtfs.py` executable on your OS. You can also directly run the script found
 in the source code. Make sure to run it with python 3.
 
-# Tests
-We use the wonderful `pytest` package for testing. Install pytest and run the tests:
+## Usage
+Run the tool over your OSM data source (or whatever osmium accepts):
+
+    python osmtogtfs.py <osmfile>
+
+After a while, depending on the file size, a file named `gtfs.zip` will be produced inside the working directory.
+Moreover, if you install the package, you will get an script called `osmtogtfs` in your python path:
+
+    $ osmtogtfs --help
+    Usage: osmtogtfs [OPTIONS] INPUT
+
+    Options:
+      --outdir PATH   Store output in this directory.
+      --zipfile PATH  Save as Zip file if provided.
+      --help          Show this message and exit.
+
+`--outdir` defaults to the working directory and if `--zipfile` is provided, the feed will be zipped and stored in
+the _outdir_ with the given name, otherwise feed will be stored as plain text in multiple files.
+
+### With Docker
+If osmium is not available in your package manager, it could be troublesome to install it manually. So here
+is an executable docker image that could be used directly. The only cavet here is passing input file to the
+docker container and also getting the results back. The containerized script will write its output to `/data`
+by default. The only step necessary is to mount the folder containing the input OSM file to `/data` inside 
+the container. The following command shows this, note that my input file is called `bremen-latest.osm.pbf` and
+is located inside `/path/to/osm` directory:
+
+    $ docker run -v /path/to/osm/:/data hiposfer/osmtogtfs /data/bremen-latest.osm.pbf
```

Comment:
@mehdisadeghi are all these paths absolute?

that is the impression that it gives me. But in most linux those paths are restricted to superuser

Reply:
Yes and No! Let me explain.

The `-v` option has two parts, left part is a path on the host (machine running docker) and the right part is an address inside the container (the ephemeral container that will be created at runtime from the image). In this case `/path/to/osm` is somewhere on the host machine and `/data` is a path inside the container. 

Then comes the second part, `/data/bremen-latest.osm.pbf`. This address is being passed to the container, and it will look internally to find it, i.e. this path should point to the filesystem inside the container.

So far it sounds that any user can run the above command... But here is the catch. At moment, in order to use docker, one should be member of `docker` group (at least on UNIX-like machines) which is root equivalent.

Reply:
@carocad I have to explain it better in the readme.